master/buildbot/scripts/logwatcher.py

```diff
@@ -131,7 +132,7 @@ def lineReceived(self, line):
             self.in_reconfig = True
 
         if self.in_reconfig:
-            print(line)
+            print(bytes2NativeString(line))
```

Comment:
well, as we are now py3 only and that line is garanteed to be byte, we can just use:

```suggestion
            print(line.decode())
```

Reply:
In that case, I should wait for #4470 to be merged before.

Reply:
I just merged https://github.com/buildbot/buildbot/pull/4552 which disables 2.7 tests (I cherry-picked that part from 4470)

Reply:
Sorry I did not have time to do this. Just saw your additional commit. :+1: